magento/magento2
# Fix multiselect with a lot of options not saved to DB

### Description (*)
Code fix to https://github.com/magento/magento2/issues/33486

### Fixed Issues (if relevant)

1. Fixes magento/magento2#33486

### Manual testing scenarios (*)

1. Create a new multiselect attribute
2. Add at least 150 options to an attribute
3. Add an attribute to the main attribute set
4. create a product and assign ALL options to the product
5. After save operation, all values should be assigned to the product

### Questions or comments

1. Who tested this feature and why it was tested on very small data set?

### Contribution checklist (*)
